packages/patternfly-4/react-core/src/components/Pagination/Navigation.tsx

```diff
@@ -96,15 +99,21 @@ export class Navigation extends React.Component<NavigationProps, NavigationState
     page: number | string,
     lastPage: number,
     onPageInput: (event: React.SyntheticEvent<HTMLButtonElement>, page: number) => void,
-    onSetPage: (event: React.SyntheticEvent<HTMLButtonElement>, page: number) => void
   ): void {
     if (event.keyCode === KEY_CODES.ENTER) {
       const inputPage = Navigation.parseInteger(this.state.userInputPage, lastPage) as number;
       onPageInput(event, Number.isNaN(inputPage) ? (page as number) : inputPage);
-      onSetPage(event, Number.isNaN(inputPage) ? (page as number) : inputPage);
+      this.handleNewPage(event, Number.isNaN(inputPage) ? (page as number) : inputPage);
```

Comment:
I wonder if there's a better way of handling that `page` here could be a string or a number. Could we instead cast it to a number in the place where it returns a string so we can avoid having to use `as number` type assertion in so many places? No worries if not, just curious in case it's an easy improvement.